pkg/controllers/user/globaldns/user_cluster_ingress.go

```diff
@@ -24,25 +24,28 @@ const (
 	appSelectorLabel          = "io.cattle.field/appId"
 	projectSelectorLabel      = "field.cattle.io/projectId"
 	UserIngressControllerName = "globaldns-useringress-controller"
+	globalNamespace           = "cattle-global-data"
```

Comment:
can we define this ns in some central place?

Reply:
done